Sources/Workspace/Workspace.swift

```diff
@@ -2435,6 +2437,7 @@ extension Workspace {
         root: PackageGraphRootInput,
         explicitProduct: String? = nil,
         forceResolution: Bool,
+        skipResolve: Bool = false,
```

Comment:
what is the difference between `skipResolve` and `forceResolution`, they seems to be related and the interaction between them is not clear to me